workers/exclusive-content/README.md

```diff
@@ -66,9 +66,9 @@ JSON object containing
 | Name | Type | Description | Example |
 | -- | -- | -- | -- |
 | paymentPointer | String | | "\$spsp.example.com/alice" |
-| nonce | String | 16 random alphanumeric characters | "ea22smqdrugcc8ty" |
+| nonce | String | base64-encoded random values generated with [crypto.getRandomValues()](https://developer.mozilla.org/en-US/docs/Web/API/Crypto/getRandomValues) | "iVBojzgBQiyIlaK2yzKUNQ==" |
 | encVerifier | String | base64-encoded AES-GCM encryption of the verifier endpoint | "nsNBmLVm11WDVbst+jJuxkA17vEH+/W6yIPeQdnoK3mdT8D/LA==" |
-| initVector | String | random initialization vector used for `encVerifier` decryption, 16 random alphanumeric characters | "6pcvwv40ckftubz8" |
+| initVector | String | base64-encoded random values generated with [crypto.getRandomValues()](https://developer.mozilla.org/en-US/docs/Web/API/Crypto/getRandomValues) | "A8S8GwC/luARiJG6xCNQBw==" |
```

Comment:
nit: I'd keep the note about this being used for encVerifier decryption

Reply:
oops, that got lost in copy pasting. Thanks.